File: signment.js

function quad(a, b, c) {
    const pos = ((-b) + Math.sqrt(Math.pow(b, 2) - 4 * a * c)) / 2 * a
    const neg = ((-b) - Math.sqrt(Math.pow(b, 2) - 4 * a * c)) / 2 * a
    const result = console.log(`x = ${pos}, x = ${neg}`);
    return result;
};

quad(1, -3, -4);

File: signment.py

import math

def quad(a,b, c):
    pos = (-(b) + math.sqrt(pow((b), 2) - 4 * a *c)) / 2 * a
    neg = (-(b) - math.sqrt(pow((b), 2) - 4 * a *c)) / 2 * a
    result = print(f"x = {pos}, x = {neg}");
    return result

quad(1, -3, -4)
    